_rules/SC1-2-Video-description-track.md

```diff
@@ -85,10 +74,6 @@ This means that the rule can only provide a pass for these succescriteria if acc
 </video>
 ```
 
```

Comment:
Include an inapplicable test case for streaming media

Reply:
This was a test case we were unable to write unfortunately. We've asked for suggestions, if you have any please let us know.